heroic-tattoo.lic

```diff
@@ -0,0 +1,57 @@
+custom_require.call(%w[common])
+
+no_pause_all
+no_kill_all
+
+no_use_rooms = get_settings.wand_watcher_no_use_rooms
+no_use_scripts = ['burgle','go2','athletics']
+downtime = DRSkill.getrank('Arcana') < 1400 ? (60 - DRSkill.getrank('Arcana')/24)*60 : 0
+timer = 3600 + downtime
+next_invoke = Time.now
+
+arg_definitions = [[
+      { name: 'once', regex: /once/i, optional: true, description: "One activation, however long that takes"},
+      { name: 'try', regex: /try/i, optional: true, description: "One ATTEMPT at activation, regardless of success or failure, and does not continue attempting"},
```

Comment:
What is the purpose of an invoke attempt that accepts a failure to invoke and doesn't care?

Reply:
T2 if folks dont want to run it all the time, but want to trigger it at the start of combat. T2 would wait for it to end, which could be a long time.